tests/i18n/test_extraction.py

```diff
@@ -758,8 +758,10 @@ class CustomLayoutExtractionTests(ExtractorTests):
     work_subdir = 'project_dir'
 
     def test_no_locale_raises(self):
-        msg = "Unable to find a locale path to store translations for file"
-        with self.assertRaisesMessage(management.CommandError, msg):
+        msg = r"Unable to find a locale path to store translations for file."
+        hint = r"HINT: Try setting your LOCALE_PATHS (default: [])"
+        with (self.assertRaisesMessage(management.CommandError, msg) and
+              self.assertRaisesMessage(management.CommandError, hint)):
```

Comment:
You don't need to check each part separately.

Reply:
Ok yeah I think was trying to find a way to check both at once but it's possible I missed something, I'll look again